typescript/utils/src/types.ts

```diff
@@ -113,5 +113,6 @@ export type ParsedLegacyMultisigIsmMetadata = {
 };
 
 export type Annotated<T> = T & {
+  chain: string; // TODO: Change to ChainName after moving to SDK from Utils
```

Comment:
bit confused by this as the transactions chain ID should always map 1:1 to a chain name

Reply:
but if we have two chains with the same chainid, we no longer know which one we're submitting to from the chainId alone

Reply:
here's an example in the duplicate-chainids pr where having the chain directly is helpful https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4599/files#diff-d572648c3e0dd7a6d4eff6670862ae1bcb5f1c1f3feb8c0d70fb60a765ae7d48

Reply:
oh I see, sorry I thought this was to different domain IDs